autoPyTorch/pipeline/components/preprocessing/image_preprocessing/padding/Pad.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any, Dict, Optional, Union
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    UniformIntegerHyperparameter,
+)
+
+import numpy as np
+
+import torch.tensor
+
+from autoPyTorch.pipeline.components.preprocessing.image_preprocessing.padding.base_pad import BasePad
+
+
+class Pad(BasePad):
```

Comment:
I see, yeah I think it makes sense to have it as an augmentation strategy, not so much as pre-processing. 

Ravin, what do you think? I am talking about 
https://github.com/LMZimmer/Auto-PyTorch_refactor/blob/08e319cc0bb1c41af376e765c6d5155924cae94e/autoPyTorch/pipeline/image_classification.py#L189

That is, instead of a fixed step in the pipeline, does it makes sense to have it as a augmentation choice?

Reply:
In case of augmentation, imgaug already has other ways of doing something like the RandomCrop plus pad(they have a crop plus keep size method). I put pad in preprocessing as if it is to be done on the images independently then it should be done as a preprocessing module(as then it will save time). I think we dont need pad as a node if wee just want the random crop plus padding functionality.

Reply:
yeah random crop plus padding sounds great, in the augmentation context